packages/hub/indexing/document-context.js

```diff
@@ -135,6 +138,40 @@ module.exports = class DocumentContext {
     return resource;
   }
 
+  async search(query) {
+    let key = `/api?${qs.stringify(query)}`;
+
+    log.debug(`Searching for records via ${key}`);
+
+    this._references.push(`${key}`);
+
+    // if (get(this, 'upstreamDoc.data.id') === id && get(this, 'upstreamDoc.data.type') === type) {
```

Comment:
@ef4 I wasn't sure what to do with this (`search` started as a copy paste of `read`). It didn't _seem_ relevant for searching so I commented it out 🤷‍♂️